common/src/legacy_memory_region.rs

```diff
@@ -28,8 +28,12 @@ pub struct LegacyFrameAllocator<I, D> {
     memory_map: I,
     current_descriptor: Option<D>,
     next_frame: PhysFrame,
+    min_frame: PhysFrame,
 }
 
+/// Start address of the first frame that is not part of the lower 1MB of frames
+const LOWER_MEMORY_END_PAGE: u64 = 0x10_000;
```

Comment:
```suggestion
/// Start address of the first frame that is not part of the lower 1MB of frames
const LOWER_MEMORY_END_PAGE: u64 = 0x10_0000;
```

Reply:
Damn. I was sure `0x10_000` was the correct hex representation for 1mb but your right. Not sure how many other places I made the same mistake :frowning: